proto/sf/substreams/intern/v2/service.proto

```diff
@@ -19,14 +19,46 @@ message ProcessRangeRequest {
 }
 
 message ProcessRangeResponse {
```

Comment:
Can we describe the update flow (each 500ms) and state and general contract of the progress message at the RPC level.

Let's add mucho comment in the proto definition please